kong/db/schema/entities/services.lua

```diff
@@ -53,19 +53,19 @@ return {
                       then_field = "path",
                       then_match = { eq = null }}},
     { conditional = { if_field = "protocol",
-                      if_match = { ne = "https" },
+                      if_match = { not_one_of = {"https", "tls"} },
```

Comment:
Would you mind updating the existing [test cases](
https://github.com/Kong/kong/blob/4b467cc70ab55612dde7db9a8e98c785d3fc49f8/spec/02-integration/03-db/02-db_core_entities_spec.lua#L1542-L1604) for db schema validation?

It's also worth having a look at the service entity unit tests in `spec/01-unit/01-db/01-schema/05-services_spec.lua` to see if there's anything that needs updating there (I realize there's some overlap between these).

Reply:
I have updated the test case description, and  I think no need to update other test cases, `spec/01-unit/01-db/01-schema/05-services_spec.lua`  have not some this schema test, schema test bad test in https://github.com/Kong/kong/blob/4b467cc70ab55612dde7db9a8e98c785d3fc49f8/spec/02-integration/03-db/02-db_core_entities_spec.lua#L1542-L1604